calico_containers/calico_ctl/utils.py

```diff
@@ -26,7 +25,6 @@
 DOCKER_ORCHESTRATOR_ID = "docker"
 NAMESPACE_ORCHESTRATOR_ID = "namespace"
 REQUIRED_MODULES = ["xt_set", "ip6_tables"]
-hostname = socket.gethostname()
```

Comment:
Is there any reason you couldn't have just changed this one line to:

```hostname = pycalico.util.get_hostname()```

or is there a reason we have to dynamically get the hostname everytime?  I assume it's not really going to work properly if the hostname changes under our feer, so I'd have thought setting up the hostname at start of day would still be sufficient.